packs/icinga2/sensors/lib/client.py

```diff
@@ -0,0 +1,136 @@
+import pycurl, json
+import urllib
+import signal
+import sys
+import time
+
+POST_PARAMS = {}
+
+class Client:
```

Comment:
Sure :) I just googled for the python examples of long polling and that was the first one I stumbled upon. That is the only reason I used py_curl :-)